editor/src/components/canvas/controls/grid-controls.tsx

```diff
@@ -1057,8 +1051,7 @@ export interface GridMeasurementHelperProps {
 }
 
 const GridMeasurementHelper = React.memo<{ elementPath: ElementPath }>(({ elementPath }) => {
-  const gridData = useGridMeasurentHelperData(elementPath)
-
+  const gridData = useGridMeasurementHelperData(elementPath)
```

Comment:
This feels overcomplicated to me. We basically dom-walk the element, collect a subset of the metadata from the computedStyle, and then recreate the styling from that with the reversal of that.
I think it would be way simpler to just copy the computedStyle values of the found grid dom element into the style props of the helper/control directly.

Reply:
So I would not use the same code in the dom-sampler and here, because here it is not necessary to have the metadata, we just need the raw computedStyle.

Reply:
We can't copy the properties over directly from one to the other as `getGridHelperStyleMatchingTargetGrid` which uses this data does do some manipulation like in the case of making the borders transparent.

There's also the `XFromProps` values which are constructed from some of the same functions as those values are passed down into the `GridResizing` component.

Reply:
Even if we don't directly copy all computedStyle, just property by property, even then it is way simpler and safer to e.g. just copy the gridTemplateRows instead of parsing/converting it to our metadata format and then print it back.

Reply:
I've done a slightly blended approach here in 0cc20b3e7. It attempts to use as much as seemed reasonable in taking the computed style directly, but still keeps a lot of the fields from this as a lot of the controls still use parsed and calculated values.